example/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM hamishmack/plutus-devcontainer
```

Comment:
Does the user need all of these files as well as the image we make? That's a bit of a shame, can we package them up somehow?

Reply:
It works without the local Dockerfile now